### PR TITLE
Twilio SMS Integration

### PR DESCRIPTION
**Pull request by KyoshiNoda**
added twilio (working)
added nodemailer (not working)
Resolves issue #24

### CODE REVIEW

File: backend/nodemailer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const nodemailer = require("nodemailer");
+
+const transporter = nodemailer.createTransport({
+  service: "hotmail",
+  auth: {
+    user: "node123@outlook.com",
+    pass: "pass",
+  },
+});
+
```
Comment (J-Obog) [inline backend/nodemailer.js]: You can put these credentials in the .env file

**Comment by J-Obog** [comment]
The `.env.sample` should contain all the keys from the `.env` except that the actual values wont be present

here's an example of a [.env.sample](https://github.com/codeunion/dotenv-example/blob/master/env.example)

**Comment by J-Obog** [comment]
Other than that it looks good, I think this PR can be focused on sms and another one can followup on the work done with the email stuff

**Comment by KyoshiNoda** [comment]
made changes on the SMS. Going to make a new PR for email API

**Review by J-Obog — APPROVED** [review]
Nice 🔥

**Pull request merged**